packages/block/test/block.spec.ts

```diff
@@ -39,44 +39,44 @@ tape('[Block]: block functions', function (t) {
   }
 
   t.test('should test transaction validation', async function (st) {
-    const block = new Block(rlp.decode(testData.blocks[0].rlp))
+    const block = Block.fromRLPSerializedBlock(testData.blocks[0].rlp)
     st.plan(2)
     await testTransactionValidation(st, block)
   })
 
   t.test('should test transaction validation with empty transaction list', async function (st) {
-    const block = new Block()
+    const block = Block.fromBlockData({})
     st.plan(2)
     await testTransactionValidation(st, block)
   })
 
   const testData2 = require('./testdata/testdata2.json')
   t.test('should test uncles hash validation', function (st) {
-    const block = new Block(rlp.decode(testData2.blocks[2].rlp))
+    const block = Block.fromRLPSerializedBlock(testData2.blocks[2].rlp)
     st.equal(block.validateUnclesHash(), true)
     st.end()
   })
 
   t.test('should test isGenesis (mainnet default)', function (st) {
-    const block = new Block()
+    const block = Block.fromBlockData({ header: { number: Buffer.from('01', 'hex') } })
```

Comment:
Note; had to add this explicit nonzero number here, we now default to block number 0 as opposed to some nonzero number in the past.